server/belga/search_providers.py

```diff
@@ -369,14 +373,34 @@ def format_list_item(self, data):
             'creditline': get_text(data['credit']),
             'source': get_text(data['source']),
             'language': get_text(data['language']),
-            'abstract': get_text(self._get_abstract(data)),
-            'body_html': self._get_body_html(data),
+            'body_html': self._get_abstract(data) + self._get_body_html(data),
```

Comment:
would be better probably to do the join here than adding `br` to abstract so it's obvious how those are connected

Reply:
I have `br` tags for `'&nbsp;&nbsp;&nbsp;&nbsp;' + get_text(self._get_newscomponent(item, 'lead')) + '<br/><br/>'` to separate abstract and body html and also body html comes with new line
for example :

            """Abstract content """
                                          -------->one Blank line
            """Body HTML content"""

Reply:
yep what I mean is it would be more readable like:
```
'body_html': self._get_abstract(data) + '<br/><br/>' + self._get_body_html(data),
```
so you can see those are connected via blank line, which is not really part of the abstract

Reply:
yep got it.